en/components/geo-map-binding-shp-file.md

```diff
@@ -7,7 +7,7 @@ mentionedTypes: ['XamGeographicMap', 'ShapefileConverter', 'Series']
 
 # Angular Binding Shape Files with Geo-spatial Data
 
-The Ignite UI for Angular map component, the [`IgxShapeDataSource`]({environment:dvApiBaseUrl}/products/ignite-ui-angular/api/docs/typescript/latest/classes/igxshapedatasource.html) class loads geo-spatial data (points/locations, polylines, polygons) from shape files and converts it to a collection of [`IgxShapefileRecord`]({environment:dvApiBaseUrl}/products/ignite-ui-angular/api/docs/typescript/latest/classes/igxshapefilerecord.html) objects.
+The Ignite UI for Angular map component, the [`IgxShapeDataSource`]({environment:dvApiBaseUrl}/products/ignite-ui-angular/api/docs/typescript/latest/classes/undefined.igxshapedatasource.html) class loads geo-spatial data (points/locations, polylines, polygons) from shape files and converts it to a collection of [`IgxShapefileRecord`]({environment:dvApiBaseUrl}/products/ignite-ui-angular/api/docs/typescript/latest/classes/undefined.igxshapefilerecord.html) objects.
```

Comment:
Oh cool there are more...
@ChronosSF @zdrawku should we bulk fix those, seems like most are the same shape DS and a couple of related

Reply:
#3990